app/webpacker/components/NewcomerChecks/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import WCAQueryClientProvider from '../../lib/providers/WCAQueryClientProvider';
+import DuplicateChecker from './DuplicateChecker';
+
+export default function Wrapper({ competitionId }) {
```

Comment:
You can move this whole component (and get rid of the whole file) into the `DuplicateChecker` definition.

Just declare the "real" `DuplicateChecker` component in there without any `export`, and then add this code snippet in there as well as the new `export default`. This is our standard practice when the Wrapper really does nothing more than adding a few contexts.

(If your wrapper has more complicated logic, like creating a reducer, then your approach of a separate file would be more justified)